### PR TITLE
Minor enhancements for cgroup metrics

### PR DESCRIPTION
**Pull request by natefoo**
- Increase the size of the `metric_value` column of `*_metric_numeric` tables to store larger values (such as those encountered in cgroup output)
- Allow the admin to choose specific cgroup params to collect

xref: #5988

### CODE REVIEW

File: lib/galaxy/model/mapping.py [hunk 1]
```diff
@@ -665,40 +665,37 @@
...
+    Column("metric_value", Unicode(model.JOB_METRIC_MAX_LENGTH)))
 
 model.JobMetricNumeric.table = Table(
     "job_metric_numeric", metadata,
     Column("id", Integer, primary_key=True),
     Column("job_id", Integer, ForeignKey("job.id"), index=True),
     Column("plugin", Unicode(255)),
     Column("metric_name", Unicode(255)),
-    Column("metric_value", Numeric(22, 7)))
+    Column("metric_value", Numeric(model.JOB_METRIC_PRECISION, model.JOB_METRIC_SCALE)))
```
Comment (dannon) [inline lib/galaxy/model/mapping.py]: Might be worth making a note where these are defined that they are used here, so people don't assume they're simple knobs that can be tweaked?

Comment (natefoo) [reply lib/galaxy/model/mapping.py]: :+1: 363c884bc05f0e8286eca01379cf8f078fbd1523

**Comment by natefoo** [comment]
Rebased.

**Comment by dannon** [comment]
@natefoo Thanks!

**Pull request merged**